app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -599,8 +599,8 @@ def hide_button?(id)
       type = get_vmdb_config.fetch_path(:server, :remote_console_type)
       return type != 'MKS' || !@record.console_supported?(type)
     when "vm_vnc_console"
-      type = get_vmdb_config.fetch_path(:server, :remote_console_type)
-      return type != 'VNC' || !@record.console_supported?(type)
+      # :remote_console_type is only useful for vmware consoles
+      return !@record.console_supported?('vnc')
```

Comment:
@himdel, what are your thoughts on keeping the comment and limiting the previous change to VMware only?  This would allow VMware vms to only have one 'VM Console' button as intended by the `:remote_console_type` while not impacting other providers.
```
      # :remote_console_type is only useful for vmware consoles
      type = get_vmdb_config.fetch_path(:server, :remote_console_type)
      return type != 'VNC' || !@record.console_supported?(type) if @record.vendor == 'vmware'
      return !@record.console_supported?('vnc')`